website/content/en/docs/Community/Contributing/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Contributing
+weight: 20
+---
+
+## Developer Certificate of Origin
+
+Every commit must be signed off with the `Signed-off-by: REAL NAME <email@example.com>` line.
+
+Use the `git commit -s` command to add the Signed-off-by line.
+
+See also <https://github.com/cncf/foundation/blob/main/dco-guidelines.md>.
+
+## Licensing
+
+Lima is licensed under the terms of [Apache License, Version 2.0](https://github.com/lima-vm/lima/blob/master/LICENSE).
+
+See also <https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md> for third-party dependencies.
+
+## Sending pull requests
+
+Pull requests can be submitted to <https://github.com/lima-vm/lima/pulls>.
+
+## Merging pull requests
+
+[Committers](../governance) can merge pull requests.
+
+A Committer shouldn't merge their own pull requests, except for trivial ones
+such as typo fixes and image updates (e.g., <https://github.com/lima-vm/lima/pull/2318>).
```

Comment:
Since I just merged my own PR I'm wondering: shouldn't this say that "committers shouldn't merge their own pull request without approval by at least one other committer"?

I don't think the merging is the issue; it is merging without independent review.

Reply:
Reworded
```markdown
## Merging pull requests

[Committers](../governance) can merge pull requests.
[Reviewers](../governance) can approve, but cannot merge, pull requests.

A Committer shouldn't merge their own pull requests without approval by at least one other Maintainer (Committer or Reviewer).

This rule does not apply for trivial pull requests such as fixing typos, CI failures,
and updating image references in templates (e.g., <https://github.com/lima-vm/lima/pull/2318>).
```